packages/device-connector/src/arduino-cli/client.ts

```diff
@@ -253,8 +291,15 @@ export class RPCClient {
       stream.destroy()
     })
 
-    stream.on('error', (err: Error) => {
+    stream.on('error', (err: StatusObject) => {
       logger.error(err)
+
+      setTimeout(() => {
+        this.initInstance().finally(() => {
+          this.boardListWatch().catch(() => {
```

Comment:
can we log?

Reply:
Done with the newest commit. This should never happen though. The lint rules just don't allow for a promise without a catch.